zone.go

```diff
@@ -15,7 +108,21 @@ func (api *API) CreateZone(z Zone) {
 	// res, err := api.makeRequest("POST", "/zones", z)
 }
 
-// ListZones liist zones on an account. Optionally takes a list of zone names
+// ZoneActivationCheck initiates another zone activation check for newly-created zones.
+// API reference:
+//   https://api.cloudflare.com/#zone-initiate-another-zone-activation-check
```

Comment:
A me oddity :-) I was originally adding multiple spaces to have it mono-spaced in godoc but I see you made that a single space after your golint commit. Gofmt doesn't seem to care about spaces in comments so didn't change it. I'll update to match yours.